From: Georges Basile Stavracas Neto <georges.stavracas@gmail.com>
Date: Sat, 19 Jun 2021 12:47:11 -0300
Subject: portal-helpers: Unref correct object

xdp_parent_free() was calling g_object_unref() at the
'data' field of _XdpParent, but the actual GObject field
is 'object'.

Unref the correct field, 'object', instead of the 'data'
field.

Origin: upstream, 0.5, commit:0fb64b35b5106947628735b141147b67d42f4497
---
 libportal/portal-helpers.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/libportal/portal-helpers.h b/libportal/portal-helpers.h
index 8b8c94f..ab14c56 100644
--- a/libportal/portal-helpers.h
+++ b/libportal/portal-helpers.h
@@ -67,7 +67,7 @@ static inline void xdp_parent_free (XdpParent *parent);
 
 static inline void xdp_parent_free (XdpParent *parent)
 {
-  g_clear_object (&parent->data);
+  g_clear_object (&parent->object);
   g_free (parent);
 }
 
